RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateReco.cc

```diff
@@ -72,7 +75,7 @@ static const int theVerboseLevel = 2;
 #include "FWCore/Utilities/interface/Exception.h"
 #else
 #include "SiPixelTemplateReco.h"
-#include "VVIObj.h"
+#include "VVIObjF.h"
```

Comment:
VVIObj is Morris's port of the original CERNLIB Fortran code.  All the variables are double.

Some time ago Vincenzo I. rewrote it to use floats instead of doubles.  It's slightly faster.  That's VVIObjF.

However, note that this change is in the *standalone* branch, which is used for standalone testing of all the templates and other pixel local reco.  (Standalone = sans CMSSW.)  So this particular change is actually a bugfix which ensures that the standalone version used for development and testing is actually the same as used in CMSSW.